app/config.py

```diff
@@ -44,6 +49,13 @@ def ADVENTURES(self):
             return self.merged_cfg['adventures']
         except:
             return dict()
+
+    @property
+    def EVENTS(self):
+        try:
+            return self.merged_cfg['events']
```

Comment:
i think it's my bad for doing `try/except` here originally, can we not do `merged_cfg.get('events', dict())`?

Reply:
yes :)